0070    javascript-functions-practice/get-character-at-index.js

```diff
@@ -0,0 +1,4 @@
+/* exported getCharacterAtIndex */
+function getCharacterAtIndex(string, index) {
+  return Array.from(string)[index];
```

Comment:
Strings also behave similarly to arrays (and have some array-like methods) so you can access the index like this `string[index]`